### PR TITLE
Updating samples after #5232.

### PR DESCRIPTION
**Pull request by ePaul**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change.
  I ran bin/run-all-petstore, but before committing I omitted some changes listed below.
- [x] Filed the PR against the correct branch: master for non-breaking changes.

### Description of the PR

No code changes, just updating the samples.

All the changes here come from #5232, which forgot to update the samples.
I omitted the changes covered by #5242, #5241, #5212, as well as changes in typescript-angular2/npm and typescript-jquery/npm, which have the date in their artifact ID (see #3084).

Interesting in the diff might be the several cases which replaced 8080 → 80.

### CODE REVIEW

**Comment by ePaul** [comment]
@wing328 some tests now fail, at least one of the tests for feign. I'll have a look at this later, if you don't want to fix this first.

```
Failed tests:   testApiClient(io.swagger.client.api.PetApiTest): expected:<.../petstore.swagger.io[]/v2> but was:<.../petstore.swagger.io[:80]/v2>
```

**Comment by ePaul** [comment]
I'm on it for the Java-based tests (because I can easily run them locally). Will push the result then for CI.

**Comment by ePaul** [comment]
These now try to start a server at port 80 (and have no permissions to do so on my computer):
* samples/server/petstore/jaxrs/jersey1
* samples/server/petstore/jaxrs/jersey2

File: samples/server/petstore/jaxrs/jersey1/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
           <stopKey>stopit</stopKey>
           <stopWait>10</stopWait>
           <httpConnector>
-            <port>8080</port>
+            <port>80</port>
```
Comment (ePaul) [inline samples/server/petstore/jaxrs/jersey1/pom.xml]: This change makes the server try to start at port 80, which (at least on normal Linux systems) is not permitted for non-privileged processes.

Comment (ePaul) [inline samples/server/petstore/jaxrs/jersey1/pom.xml]: See #5246 for a possible fix for that.

**Comment by ePaul** [comment]
I merged the branch of #5246 into this one, so that should be merged first, if it is deemed a plausible solution. (I can rebase this one afterwards, or revert the changes if not deemed okay.)

**Comment by wing328** [comment]
@ePaul thanks for fixing all those test cases.

**Pull request merged**